src/program.ts

```diff
@@ -50,6 +56,12 @@ Links:
 
       spinner.succeed('ASCII Tree generated');
 
+      if(options.autoCopy) {
```

Comment:
Since we defined the option as `-c, --copy` the if statement will always evaluate to `false`, we'd have to modify it:
```suggestion
      if(options.copy) {
```